bitrise.yml

```diff
@@ -305,7 +305,7 @@ workflows:
         title: Set Bundler to use local vendor directory
     - git-clone@6:
         inputs:
-        - clone_depth: '100'
+        - merge_pr: 'no'
```

Comment:
I looked at the definition here:
https://github.com/bitrise-steplib/steps-git-clone/blob/master/step.yml

I suspect i don't fully understand how this works, but i would have expected "merge_pr" to be set to "yes" since we would want to merge the pr into the checked out branch and run a size check on it?

Reply:
merge_pr set to `yes` creates a new repo, fetches the base ref, then merges the PR branch on top of it. When it's set to `no`, it just checks out the PR branch directly. I set it to `no` for this job because we do some repo manipulation in the size_report script, and it seemed like whatever they were doing was getting in my way. But maybe adding the full 'git fetch origin' was enough to get the refs I need. ¯\_(ツ)_/¯